Library/Models/TableViewDataSource.swift

```diff
@@ -100,9 +100,9 @@ class TableViewDataSource<Item: TableItem> {
                             }
                         }
                     }
-
-                    self.delegate?.dataSourceDidChangeContent()
                 }
+
+                defer { self.delegate?.dataSourceDidChangeContent() }
```

Comment:
Hmm, now I'm not so sure. 💭 What do you have in mind? Just removing defer and calling source did change directly?